packages/matter.js/src/cluster/SwitchCluster.ts

```diff
@@ -0,0 +1,78 @@
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 1.11
+ */
+export const SwitchCluster = Cluster({
+    id: 0x3b,
+    name: "Switch",
+    revision: 1,
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.11.4 */
+    features: {
+        /**  Indicates a switch that maintains its position after being pressed (or turned) */
+        latchingSwitch: BitFlag(0),
+
+        /** Indicates a switch that does not maintain its position after being pressed (or turned)  */
+        momentarySwitch: BitFlag(1),
+
+        /** Indicates a momentary switch that can distinguish and report release events. */
+        momentarySwitchRelease: BitFlag(2),
+
+        /** Indicates a momentary switch that can distinguish and report long presses from short presses. */
+        momentarySwitchLongPress: BitFlag(3),
+
+        /** Idicates a momentary switch that can distinguish and report double press 
+         *  and potentially multiple presses with more events, such as triple press, etc. 
+         */
+        momentarySwitchMultiPress: BitFlag(4),
+    },
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.11.5 */
+    attributes: {
+        /** Indicates the maximum number of positions the switch has with a minimum of 2 */
+        numberOfPositions: Attribute(0, TlvUInt8.bound({ min: 2 }), { default: 2 }),
+
+        /** Indicates the position of the switch with a valid range of zero to NumberOfPositions-1 */
+        currentPosition: Attribute(1, TlvUInt8, { default: 0 }),
+
+        /** Indicates how many consecutive presses can be detected/reported by a multi-press switch */
+        multiPressMax: OptionalAttribute(2, TlvUInt8.bound({ min: 2 }), { default: 2 }),
+    },
+
+    /** @see {@link MatterCoreSpecificationV1_0} § 11.11.7 */
+    events: {
+        /** Generated when the latching switch is moved to a new position. */
+        SwitchLatched: OptionalEvent(0, EventPriority.Info, { newPosition: TlvField(0, TlvUInt8) }),
+
+        /** Generated when the momentary switch starts to be pressed. */
+        InitialPress: OptionalEvent(1, EventPriority.Info, { newPosition: TlvField(0, TlvUInt8) }),
+
+        /** Generated when the momentary switch has been pressed for a "long" time. */
+        LongPress: OptionalEvent(2, EventPriority.Info, { newPosition: TlvField(0, TlvUInt8) }),
+
+        /** Generated when the momentary switch has been released. */
+        ShortRelease: OptionalEvent(3, EventPriority.Info, { previousPosition: TlvField(0, TlvUInt8) }),
+
+        /** Gnerated when the momentary switch has been released. */
+        LongRelease: OptionalEvent(4, EventPriority.Info, { previousPosition: TlvField(0, TlvUInt8) }),
+
+        /** Indicates how many times the switch has been pressed in a multi-press sequence during that sequence */
+        MultiPressOngoing: OptionalEvent(5, EventPriority.Info, { newPosition: TlvField(0, TlvUInt8), currentNumberOfPressesCounted: TlvField(1, TlvUInt8) }),
```

Comment:
currentNumberOfPressesCounted has min 2 as bound

Reply:
why you used min in quotes?

Reply:
I don't know :-)  Will fix both!